qadence_embeddings/callable.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import annotations
+
+from importlib import import_module
+from logging import getLogger
+from typing import Callable
+
+from numpy.typing import ArrayLike
+
+logger = getLogger(__name__)
+
+ARRAYLIKE_FN_MAP = {
```

Comment:
```suggestion
ARRAYLIKE_CONSTRUCTOR_MAP = {
```

Or something like this ?